Renaming Use, Migrate, MST and troubleshooting (#4193)
timescale/docs: use-timescale/services/service-management.md

data discrepancy between $SERVICE_SHORTs.

</Highlight>

<Procedure>

### Forking a $SERVICE_SHORT

1.  In $CONSOLE_LONG, from the `Services` list, ensure the $SERVICE_SHORT
    you want to form has a status of `Running` or `Paused`, then click the name
    of the $SERVICE_SHORT you want to fork.
1.  Navigate to the `Operations` tab.
1.  In the `Service management` section, click `Fork service`. In the dialog,
    confirm by clicking `Fork service`. The forked $SERVICE_SHORT takes a few minutes
    to start.
1.  [](#)<Optional />To change the configuration of your fork, click
    `Advanced options`. You can set different compute and storage options,
    separate from your original $SERVICE_SHORT.
1.  Confirm by clicking `Fork service`. The forked $SERVICE_SHORT takes a few minutes
    to start.
1.  The forked $SERVICE_SHORT shows in the `Services` dashboard with a label stating
    which $SERVICE_SHORT it has been forked from.

<img
class="main-content__illustration"
width={1375} height={944}
src="https://assets.timescale.com/docs/images/tsc-forked-service.webp"
alt="Fork a Tiger Cloud service"
/>

</Procedure>

> ### Reset $SERVICE_SHORT password

Reviewer: ```suggestion
### Reset your $SERVICE_SHORT password
```

You can reset your $SERVICE_SHORT password from the `Operations` dashboard. This is the

> password you use to connect to your $SERVICE_SHORT, not the password to your $ACCOUNT_LONG. To reset your $ACCOUNT_SHORT password, navigate to the `Account` page.

Reviewer: ```suggestion
password you use to connect to your $SERVICE_SHORT, not the password for $CONSOLE. To reset your $CONSOLE_SHORT password, navigate to the `Account` page.
```

> When you reset your $SERVICE_SHORT password, you are prompted for your $ACCOUNT_SHORT password. When you have authenticated, you can create a new $SERVICE_SHORT password,

Reviewer: ```suggestion
When you reset your $SERVICE_SHORT password, you are prompted for your $CONSOLE_SHORT password. When you have authenticated, you can create a new $SERVICE_SHORT password,
```

ask $CONSOLE_SHORT to auto-generate a password, or switch your authentication
type between SCRAM and MD5.

SCRAM (salted challenge response authentication mechanism) and MD5 (message
digest algorithm 5) are cryptographic authentication mechanisms. $CONSOLE_LONG
uses SCRAM by default. It is more secure and strongly recommended. The MD5
option is provided for compatibility with older clients.

> ### Pause $SERVICE_SHORT

Reviewer: ```suggestion
### Pause a $SERVICE_SHORT
```
